core/tests/protractor_utils/player.js

```diff
@@ -134,6 +134,16 @@ var expectExplorationToNotBeOver = function() {
   )).last().getText()).not.toEqual('Congratulations, you have finished!');
 };
 
+var openFeedbackPopup = function() {
+  element(by.css('.protractor-test-exploration-feedback-popup-link')).click();
+};
+
+var submitFeedback = function(feedback) {
```

Comment:
Perhaps you should incorporate openFeedbackPopup() as part of this function. I think it makes sense to regard each function in this file as "starting" from the generic learner page, and this assumes that the feedback popup is already open.